book/src/example.md

```diff
@@ -3,136 +3,26 @@
 > ## *This page is out-of-date and under active development.*
 
 The
-[HostPath](https://github.com/kubernetes-csi/drivers/tree/master/pkg/hostpath)
-can be used to provision local storage in a single node test. This
-section shows how to deploy and use that driver in Kubernetes.
+[HostPath](https://github.com/kubernetes-csi/csi-driver-host-path)
```

Comment:
"The Hostpath CSI driver is a simple sample driver that provisions a directory on the host. It can be used as an example to get started writing a driver, however it is not meant for production use."

Reply:
yes, I will replace the sentence, and overall I think the first part can be simplified. The header "Deployment" should go away, and remaining sentences get reduced and for one chapter

Reply:
it may be better idea to link to deploy/ directory instead of deployment script, as the whole set of files under deploy/ forms this example